README.rst

```diff
@@ -25,7 +25,7 @@ Molecule Inspec Plugin
    :target: LICENSE
    :alt: Repository License
 
-Molecule Inspec Plugin is designed to allow used of inpect as a verifier.
+Molecule Inspec Plugin is designed to allow used of inpec as a verifier.
```

Comment:
Hey @kclinden could you update this to "Inspec"?

Reply:
lol typo in my typo fix :) Resolved